canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -883,6 +883,18 @@ Cypress.Commands.add("verifyLeftPanelWidth", (width) => {
 	});
 });
 
+Cypress.Commands.add("verifyLeftFlyoutPanelWidth", (width) => {
+	cy.get(".left-flyout-panel").should((element) => {
+		compareCloseTo(element[0].offsetWidth, width);
+	});
+});
+
+Cypress.Commands.add("verifyLeftFlyoutPanelHeight", (height) => {
+	cy.get(".left-flyout-panel").should((element) => {
+		compareCloseTo(element[0].offsetHeight, height);
+	});
+});
+
```

Comment:
I changed the issue description to make this work item to add similar tests for the right-flyout as well.  I thought we already had tests for the right-flyout but looks like we don't. You'll need to create a right-flyout.cy.js file for it (and add a copyright to it).